### PR TITLE
Update Issue Templates

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

 - GitHub offers a [new issue template builder](https://help.github.com/en/enterprise/2.20/user/github/building-a-strong-community/about-issue-and-pull-request-templates#issue-templates). We can create separate issue templates for "Feature requests" and "Bug reports".
- Maybe we can trial this in one repo to see if we like it. Let me know your thoughts!

#### Sample photo
Here is a sample of what this would look like for merchants (except we would only have 2 issue templates instead of a "custom" option).

Screen Shot 2020-03-13 at 10 30 28 AM

 ### Checklist
 ~- [ ] Added a changelog entry~

### Authors
- @scannillo

### CODE REVIEW

**Comment by sestevens** [comment]
I like it!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (crookedneighbor) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should the about include a note about reaching out to support for help with your integration?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Feature request
+about: Suggest an idea for our SDK
```
Comment (crookedneighbor) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Same note here regarding the about.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (scannillo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: The "about" section here is what gets shown in the sub-label for the issues drop down. I think adding the note about contacting support might be too much text for that view. 

issue

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. See error
+
+**Expected behavior**
+Description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (hollabaq86) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
If applicable, add screenshots to help explain your problem (for security DO NOT include credit card numbers).
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Only open an issue here if you think you've found an issue with our SDK. If you need help troubleshooting your integration, reach out to Braintree Support at https://help.braintreepayments.com. -->
+
+**Integration Details (please complete the following information):**
+ - SDK/Library Version: [e.g. 4.7.2]
```
Comment (sestevens) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - SDK/Library Version: [e.g. 8.0.0]
```

Just since 4.7.2 is pretty old now 😃

**Pull request merged**